src/init.cpp

```diff
@@ -148,6 +156,11 @@ void updateConfiguration(pika::program_options::variables_map const& vm, configu
                            "UMPIRE_DEVICE_MEMORY_POOL_INITIAL_BYTES",
                            "umpire-device-memory-pool-initial-bytes");
   cfg.mpi_pool = (pika::resource::pool_exists("mpi")) ? "mpi" : "default";
+
+  // update tune parameters
+  auto& param = getTuneParameters();
```

Comment:
The usage of `configuration` and `tuneparams` is different:
- `configuration` is set up and used in the initialization of dlaf. Any change to the values after the setup are irrelevant.
- `tuneparams` is set up from env variables and command line parameters. However they are used during the scheduling of algorithms, and users can change it to affect the next calls.

Reply:
I see. I'd be slightly against an interface where the tuneparams is "state" that the user can change after initialization, especially with possible concurrent calls to algorithms. I'd rather treat `tuneparams` as a default which is never changed after initialization (i.e. exactly as `configuration`, and hence I imagined it could simply be part of `configuration`), but that can be overridden explicitly in the algorithm calls that need it. Does that make sense? I may be missing how you're imagining that the band size would be changed so take this with a grain of salt.

Reply:
I know that there might be some critical points. I do not expect users to use it unless they know what they are doing.
On the other hand, it simplifies writing tests a lot.

Reply:
Ok, fair enough! As long as you're aware of and ok with the tradeoff I'm ok with it.